test/CAS/cas-explicit-module-map.swift

```diff
@@ -33,13 +41,15 @@
 // RUN: llvm-cas --cas %t/cas --make-node --data %t/kind.blob @%t/String.key @%t/schema.casid > %t/String.casid
 // RUN: llvm-cas --cas %t/cas --put-cache-key @%t/String.key @%t/String.casid
 
-// RUN: llvm-cas --cas %t/cas --make-blob --data %t/SwiftShims.pcm | tr -d '\n' > %t/Shims.key
-// RUN: llvm-cas --cas %t/cas --make-node --data %t/kind.blob @%t/Shims.key @%t/schema.casid > %t/Shims.casid
-// RUN: llvm-cas --cas %t/cas --put-cache-key @%t/Shims.key @%t/Shims.casid
+/// llvm-cas --cas %t/cas --make-blob --data %t/SwiftShims.pcm | tr -d '\n' > %t/Shims.key
```

Comment:
Disabled test code?

Reply:
Nice catch. That was the old way of setup cache key. Removing.